woocommerce/wc-admin
# Refactor payments to allow management of methods

Fixes (partially) #6699.

Note that `Setup required` will be handled in a follow-up and separation of methods is being handled in https://github.com/woocommerce/woocommerce-admin/pull/6766. cc @joelclimbsthings if you need me to make any changes to this in prep for 6766 please let me know.

This PR refactors the payments screen into individual functional components and adds `Set up/Manage/Enable` links.

### Screenshots

<img width="702" alt="Screen Shot 2021-04-12 at 4 36 09 PM" src="https://user-images.githubusercontent.com/10561050/114464362-a84cf880-9bb3-11eb-918c-ba7bff0b39a3.png">


### Detailed test instructions:

1. Do not select "CBD industry" as a store industry during onboarding.
2. Make various payment methods visible by switching to different countries.
3. Attempt to set up various payment methods.
4. Make sure that after setup, a `Manage` link is shown that links to the payment method settings page.
5. Check that simple methods like, cash delivery or bank transfer initially have an `Enable` option.
